broker/setup_broker/swift/create_vm.sh

```diff
@@ -2,15 +2,17 @@
 # Creates or deletes the GCP VM instances needed by the broker.
 # This script will not delete VMs that are in production
 
-
-broker_bucket=$1 # name of GCS bucket where broker files are staged
+# name of GCS bucket where broker files are staged
+gcs_broker_bucket=$1
+# "False" uses production resources
+# any other string will be appended to the names of all resources
 testid="${2:-test}"
-#   "False" uses production resources
-#   any other string will be appended to the names of all resources
-teardown="${3:-False}" # "True" tearsdown/deletes resources, else setup
-survey="${4:-swift}"
+# "True" tearsdown/deletes resources, else setup
+teardown="${3:-False}"
 # name of the survey this broker instance will ingest
+survey="${4:-swift}"
 zone="${5:-us-central1-a}"
+project_id="${6:-PROJECT_ID}"
```

Comment:
No need to define yet another name for this variable. Just use `$PROJECT_ID` directly where applicable. (GCP makes this variable confusing by using different names for it in different contexts -- PROJECT_ID and GOOGLE_CLOUD_PROJECT being most common, but there's at least one more. In our scripts we try to standardize on PROJECT_ID.)